config/routes.rb

```diff
@@ -9,7 +9,9 @@
   post '/slack/install' => 'slack#early_access', as: :early_access
 
   get '/login' => redirect('/auth/slack'), as: :login
-  post '/logout' => 'sessions#destroy', as: :logout
+
+  get '/logout' => 'sessions#destroy', as: :logout
```

Comment:
The reason we use POST for logouts is to mitigate against CSRF. You can change the link in the header to submit as a post:

```erb
<%= link_to 'Logout', logout_path, method: :post %>
```

Reply:
Cool, yeah a CSRF that logs a user out could be annoying. Fix this.